OpenEdX/Managers/PushNotificationsManager/Listeners/FCMListener.swift

```diff
@@ -6,11 +6,28 @@
 //
 
 import Foundation
+import Swinject
+import FirebaseMessaging
 
 class FCMListener: PushNotificationsListener {
+    
+    private lazy var deepLinkManager: DeepLinkManager? = {
+        Container.shared.resolve(DeepLinkManager.self)
+    }()
+    
     // check if userinfo contains data for this Listener
     func shouldListenNotification(userinfo: [AnyHashable: Any]) -> Bool {
         let data = userinfo["gcm.message_id"]
         return userinfo.count > 0 && data != nil
     }
+    
+    func didReceiveRemoteNotification(userInfo: [AnyHashable: Any]) {
+        // With swizzling disabled you must let Messaging know about the message, for Analytics
+        Messaging.messaging().appDidReceiveMessage(userInfo)
```

Comment:
I guess it should move below the guard statement as the notification can be from the braze.

Reply:
Yeap ✅